src/sql/workbench/services/dialog/browser/dialogPane.ts

```diff
@@ -47,6 +48,10 @@ export class DialogPane extends Disposable implements IThemable {
 		super();
 	}
 
+	public get pageNumberDisplayText(): string {
+		return localize('wizardPageNumberDisplayText', "Step {0}", this.pageNumber);
```

Comment:
As an added bonus this is now localized as well which was something I remember thinking to myself we should fix at some point in the future 😁